apps/examples/testcase/le_tc/kernel/tc_clock.c

```diff
@@ -169,7 +169,7 @@ static void tc_clock_clock_gettimeofday(void)
 
 static void tc_clock_clock_timer(void)
 {
-	systime_t itime = g_system_timer;
+	clock_t itime = g_system_timer;
```

Comment:
Let's remove this tc. The clock_timer is an internal API. This should not be called from kernel outside.